rules_building_block/defense_evasion_dotnet_clickonce_dfsvc_netcon.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2023/09/25"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/09/25"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of DotNet ClickOnce installer via Dfsvc.exe trampoline. Adversaries may take advantage of
+ClickOnce to proxy execution of malicious payloads via trusted Microsoft processes.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Execution via Microsoft DotNet ClickOnce Host"
+risk_score = 21
+rule_id = "5297b7f1-bccd-4611-93fa-ea342a01ff84"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Rule Type: BBR", "Data Source: Elastic Defend"]
+building_block_type = "default"
+type = "eql"
+
+query = '''
+sequence by user.id with maxspan=5s
+ [process where event.action == "start" and
+  process.name : "rundll32.exe" and process.command_line : ("*dfshim*ShOpenVerbApplication*", "*dfshim*#*")]
+ [any where event.category : ("network", "dns") and process.name : "dfsvc.exe"]
+'''
```

Comment:
Curious about this one, would `process.entity_id` / `process.parent.entity_id` work here? And instead of the any category, use `network`?

Reply:
Good catch, dns is an endpoint category, I will fix both issues. Thanks!

Reply:
Confirmed that network events in Windows don't have process.parent.entity_id information, the sequence will need to use user.id here, or host.id